### ISSUE TITLE
Fixing Tag and Implementing a Graphical Visualization tool

### ISSUE THREAD

**Issue by kx06** [acmpesuecc/myGit_hn6#4]
Fix tag not being implemented fully.

"@" is currently the alias for HEAD, rewrite log and tag such that they optionally accept oid and if that isn't provided it defaults to HEAD. 

gitk  is a graphical visualization tool for Git, implement something similar for mygit, our command to run the tool will be `mygit k`

**Comment by Alex-Hunterz** [comment]
Can i be assigned??

**Comment by kx06** [comment]
!assign @Alex-Hunterz  60

**Comment by kx06** [comment]
!assign @Alex-Hunterz

**Comment by Alex-Hunterz** [comment]
sorry i stepped away for dinner... could you extend the current timer...

**Comment by kx06** [comment]
yes, ill deassign and reassign this issue to you but for 60ish minutes, make sure you eat dinner well and then hopefully a PR

**Comment by kx06** [comment]
!deassign

**Comment by kx06** [comment]
!assign @Alex-Hunterz

**Comment by Alex-Hunterz** [comment]
Thanks a lot

**Comment by Alex-Hunterz** [comment]
okay ummm im done with my changes.. can i try a pr?

**Comment by kx06** [comment]
hello @Alex-Hunterz  sorry I didnt know you put a message, do you still wanna try a pr ?

**Comment by Alex-Hunterz** [comment]
yeah just give me a min... i seem to have added my venv files as well when commiting... let me revert back the changes... and try a pr

**Comment by kx06** [comment]
ok please let me know in which sem hall you are currently seated.

**Comment by Alex-Hunterz** [comment]
im in sem hall 2

**Comment by kx06** [comment]
!deassign

**Comment by kx06** [comment]
!assign @Alex-Hunterz 60

**Comment by Alex-Hunterz** [comment]
Hey man.... I can't seem to crack this... so maybe you can deassign it... ill try it out later

**Comment by kx06** [comment]
Yea sure

**Comment by kx06** [comment]
!deassign

**Comment by AutisticCoder-9000** [comment]
Can I get assigned for this?

**Comment by kx06** [comment]
!deassign

**Comment by kx06** [comment]
!assign @AutisticCoder-9000 60

**Comment by kx06** [comment]
@AutisticCoder-9000 are your alr assigned to something else ?

**Comment by AutisticCoder-9000** [comment]
Yeah, but I'm done for one of them

**Comment by kx06** [comment]
Then ask the maintainer to deassign you from those issues.

**Comment by AutisticCoder-9000** [comment]
ok

**Comment by AyushChakraborty** [comment]
please assign this to me

**Comment by sampada115** [comment]
can you assign me this issue? @kx06

**Comment by kx06** [comment]
!deassign

**Comment by kx06** [comment]
!assign @sampada115 60

**Comment by kx06** [comment]
@sampada115 

Are u assigned to some other issue ?

**Comment by sampada115** [comment]
no

**Comment by sampada115** [comment]
i'm sorry i'm assisgned just checked now can you deassign

**Comment by sampada115** [comment]
@kx06

**Comment by kx06** [comment]
!deassign

**Comment by kx06** [comment]
!assign @sampada115 60

**Comment by sampada115** [comment]
@kx06 i need some of your help where can i approach you?

**Comment by kx06** [comment]
Room 002
